crates/oxc_linter/src/snapshots/empty_tags.snap

```diff
@@ -0,0 +1,75 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: empty_tags
+---
+  ⚠ eslint-plugin-jsdoc(empty-tags): Expects the void tags to be empty of any content.
+   ╭─[empty_tags.tsx:3:17]
+ 2 │                       /**
+ 3 │                        * @abstract extra text
+   ·                          ─────────
+ 4 │                        */
+   ╰────
+  help: `@abstract` tag should be empty.
```

Comment:
"The tag should be empty" is confusing. Maybe something like "the description text will not show up in jsdoc?" to convey it's useless What the proper term for the text anyway?

Reply:
> "the description text will not show up in jsdoc?"

This may be tricky topic. 😓 

- Original JSDoc.app does not expect THIS for `@async` and may just ignore in its implementation?
  - https://jsdoc.app/tags-async
  - https://github.com/jsdoc/jsdoc/blob/7565dfd53cb5d86e334a9a3fa1193c767a31a0d7/packages/jsdoc-tag/lib/definitions/core.js#L59
- But TypeScript renders THIS( and seems to accept THIS everywhere)
  - ![image](https://github.com/oxc-project/oxc/assets/6259812/db901c69-4629-48cb-a7c6-576320a7dd2b)
  - ![image](https://github.com/oxc-project/oxc/assets/6259812/3ce4e6fd-59ff-4d27-8e9d-ad2f0411186e)

> What the proper term for the text anyway?

I call THIS as "comment part for tag" or just "tag comment" since TS defined THIS in `.comment` property.

Original JSDoc seems to use the term "value"?, `jsdoccomment`(and `comment-parser`) using the term "description".

Reply:
Tag comment sounds good to me.

If there are things can render this comment ... then we need to put this into the restriction or pedantic category.